Hooks.php

```diff
@@ -62,6 +68,15 @@ public static function onParserFirstCallInit(Parser &$parser) {
     }
   }
 
+ /*   public static function onParserFirstCallInit(Parser &$parser) {
+        global $wgGoogleRichCardsAnnotateBooks;
+        
+        if($wgGoogleRichCardsAnnotateBooks) {
+            $book = Event::getInstance();
+            $parser->setHook('book', [$book, 'parse']);
+        }
+    }*/
```

Comment:
This normally should be uncommented. The the same function using Event class(above) **must** not exist.

Reply:
Uncomment this lead to crash

Reply:
It should work properly after fixing other issues.